client/run.go

```diff
@@ -290,6 +303,67 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 	return string(jsonString), nil
 }
 
+func parseVolume(volStr string) (*pod.UserVolume, *pod.UserVolumeReference, error) {
```

Comment:
won't update this. got this puzzle if I don't use pointer (): https://www.reddit.com/r/golang/comments/2xmnvs/returning_nil_for_a_struct/